src/main/java/cc/aabss/eventutils/UpdateChecker.java

```diff
@@ -35,36 +36,52 @@ private void notifyUpdate(@NotNull String latestVersion) {
     }
 
     public void checkUpdate() {
+        if (!EventUtils.MOD.config.updateChecker
+                || Versions.MC_VERSION == null
+                || Versions.EU_VERSION == null
+                || Versions.EU_VERSION_SEMANTIC == null) return;
+
+        // Get client
+        final MinecraftClient client = MinecraftClient.getInstance();
+        if (client.player == null) return;
+
+        final HttpClient httpClient = HttpClient.newHttpClient();
+
         try {
```

Comment:
wanted to keep everything in try statement just in-case something goes wrong
doesnt matter if update checker fails